src/mono/wasm/debugger/DebuggerTestSuite/EvaluateOnCallFrame2Tests.cs

```diff
@@ -731,5 +731,19 @@ public async Task EvaluateLocalObjectFromAssemblyNotRelatedButLoaded()
                     ("dt+1", "Cannot evaluate '(dt+1\n)': (2,9): error CS0019: Operator '+' cannot be applied to operands of type 'object' and 'int'")
                 );
            });
+
+        [Fact]
+        public async Task EvaluateMethodsOnEnum() => await CheckInspectLocalsAtBreakpointSite(
```

Comment:
IIUC, this PR is fixing the case of a type having a `ToString` override with parameters, and isn't specific to enums as such. So, essentially we are trying to invoke the no-params overload to `ToString`.

Reply:
I think you are correct. We had other tests with `ToString`:
https://github.com/dotnet/runtime/blob/5883b728c8b29a7026cf3a807a22928cee844bc5/src/mono/wasm/debugger/DebuggerTestSuite/EvaluateOnCallFrame2Tests.cs#L582
https://github.com/dotnet/runtime/blob/5883b728c8b29a7026cf3a807a22928cee844bc5/src/mono/wasm/debugger/DebuggerTestSuite/EvaluateOnCallFrame2Tests.cs#L613
but all of them are no-param method.
The name of test is correct, though and the PR proves that calling methods on enums is possible, so it really fixes the linked issue. I will update the PR's name.